lib/wallet/walletdb.js

```diff
@@ -2383,6 +2386,9 @@ class WalletOptions {
       this.wipeNoReally = options.wipeNoReally;
     }
 
+    assert(options.honey != null);
```

Comment:
Why is this unlike the others? It would be keep to keep this code functional without honeycomb set up